BatterySensor.h

```diff
@@ -388,7 +411,12 @@ class IrqInternalBatt {
   static void irq () {
     __gb_BatCount++;
     if( __gb_BatCount > 10 ) { // ignore first 10 values
-      uint16_t v = 1100UL * 1024 / ADC;
+
+      uint16_t v =(m_SensPin > 0) ?
+          1100UL * 57UL * ADC / 1024 / 10
```

Comment:
57UL ist vom Spannungsteilerverhältnis abhängig. Sollte man das nicht besser einstellen können?

Reply:
Ja, das kann ich noch als letzten Parameter mit in die `initExternal()` packen 👍

Reply:
750909a